### ISSUE TITLE
The framenumber counter isn't necessarily equal to the output frame number

### ISSUE THREAD

**Issue by lorenzhs** [cebe/pdfpc-latex-notes#21]
For example, our corporate template starts with `framenumber` 0 for the title page. I don't know if it's possible to figure this out automatically, but being able to pass a known offset as a package option would be good.

As a workaround, I inserted the following line before the definition of `pnote`, and replaced `theframenumber` with `pdfpcframenumber`:
```latex
\newcommand{\pdfpcframenumber}{\the\numexpr\value{framenumber}+1\relax}
```
That is completely unportable, of course, but could just as well be applied with an offset that defaults to 0.

**Comment by cebe** [comment]
> For example, our corporate template starts with framenumber 0 for the title page.

could you explain why this is the case?

**Comment by lorenzhs** [comment]
I think the idea is that the first content slide is slide 1 and the title doesn't have a visible frame number. But this applies equally to all other crazy things people might do with the `framenumber` counter.

**Comment by lorenzhs** [comment]
Uhm, more generally, isn't `framenumber` the wrong counter when using overlays, or is this another pecularity in our corporate template?

**Comment by cebe** [comment]
well, current implementation is working well with overlays when using plain beamer package without adjustments, so I assume it is correct. The expected behavior is to have notes per frame and not per overlay.

**Comment by lorenzhs** [comment]
Yeah I agree that makes sense, it's probably our template that's broken. Switching to the `page` counter fixes it though :man_shrugging: - Thanks.
